WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -30,13 +36,27 @@ struct POSSendReceiptView: View {
                 .font(POSFontStyle.posTitleRegular)
                 .focused()
                 .padding()
+            if let errorMessage = errorMessage {
+                Text(errorMessage)
+                    .font(POSFontStyle.posBodyRegular)
+                    .foregroundColor(.red)
```

Comment:
As I understand the UI is not fully built yet, and it's a feature in progress.

A couple of observations from view behavior, you may already have ideas about it, it's all optional:
-The floating button goes up with a keyboard. It may be the first time we use user input on POS. It would be good to clarify if that's what we want.
- The button is in a static position and keeps in this place even when a keyboard is dismissed. I wonder if it should be at the bottom and raised together with the keyboard.
- A button moves down when an error appears. Likely would look okay with a little animation.

https://github.com/user-attachments/assets/7737d192-99c7-460c-a033-e406576cd610

Reply:
Yes, at the moment we're yolo'ing a bit based on the prototypes. I'm adding the behaviour for the floating button, and the resting position of the sending button as something to follow-up with design.

> A button moves down when an error appears. Likely would look okay with a little animation.

Good catch, updated https://github.com/woocommerce/woocommerce-ios/pull/14651/commits/af3c26ac14af45477f48ea0a68ab3ebf41caa0ba